scapy/route.py

```diff
@@ -178,17 +178,22 @@ def route(self, dst=None, verbose=conf.verb):
         return ret
 
     def get_if_bcast(self, iff):
+        bcast_list = []
         for net, msk, gw, iface, addr, metric in self.routes:
             if net == 0:
-                continue
+                continue    # Ignore default route "0.0.0.0"
+            elif ltoa(msk) == "255.255.255.255":
```

Comment:
Sorry I missed that but we don't need to compute `ltoa()` for a static comparison:
```suggestion
            elif msk == 0xffffffff:
```

Reply:
Sensible optimisation - sorry I didn't think of this in the first place and thanks for catching it!